test/uk/gov/hmrc/cgtpropertydisposalsfrontend/services/UKAddressLookupServiceImplSpec.scala

```diff
@@ -297,21 +297,21 @@ class UKAddressLookupServiceImplSpec extends AnyWordSpec with Matchers with Mock
             List(
               UkAddress(
                 "line1",
-                Some("line2"),
```

Comment:
did this spec need to change at all? why are the addresses used here being updated?

Reply:
So effectively, the addresses in the list are not updated but arranged in sorted order.